docs/gl_objects/groups.rst

```diff
@@ -458,3 +458,24 @@ Edit group push rules::
 Delete group push rules::
 
     pr.delete()
+
+Group Service Account
+======================
```

Comment:
I think pre-commit will complain if we don't match the RST title here:

```suggestion
Group Service Account
=====================
```

Reply:
Thanks! I sort it out. I did run pre-commit on this change and it didn't pick this issue. I'll verify that too.